flow/activities/flowable.go

```diff
@@ -78,6 +78,13 @@ func (a *FlowableActivity) SetupMetadataTables(ctx context.Context, config *prot
 	}
 	defer connectors.CloseConnector(ctx, dstConn)
 
+	// this should have been done by DropFlowDestination
+	// but edge case due to late context cancellation
+	if err := dstConn.SyncFlowCleanup(ctx, config.FlowName); err != nil {
```

Comment:
can e2e test this with a drop test that passes false for deleting stats

Reply:
can be done in #2463